src/backend/aspen/workflows/nextstrain_run/run_nextstrain_scheduled.sh

```diff
@@ -30,8 +30,6 @@ TEMPLATE_ARGS=$(jq -c . < "${TEMPLATE_ARGS_FILE}")
 
 workflow_id=$(aspen-cli db create-phylo-run                                                                           \
                   --group-name "${GROUP_NAME}"                                                                               \
-                  --all-group-sequences                                                                                      \
-                  --builds-template-file "/usr/src/app/aspen/workflows/nextstrain_run/builds_templates/${TEMPLATE_FILENAME}" \
                   --builds-template-args "${TEMPLATE_ARGS}"                                                                  \
                   --tree-type "${TREE_TYPE}"
 )
```

Comment:
Per [here](https://github.com/chanzuckerberg/aspen/blob/67aff9785bc856ab20826f9ac2bc56ecde5c5472/src/backend/aspen/workflows/nextstrain_run/builds_templates/mega_template.yaml#L26), we need to generate an empty `/ncov/data/include.txt` in this script

Reply:
> 

Thanks, will do!